HelpSource/Classes/LFDNoise3.schelp

```diff
@@ -109,17 +86,13 @@ SynthDef( "canto_revDelay", { |in = 0, private = 0, amp = 0.1,
 	});
 
 	out = out.collect({ |sig, i|
-		RoundPan.ar( ~canto.numChan ? 4, sig, LFDNoise3.kr(0.05, 2) );
-	}).sum;
-
+		Pan4.ar(sig, trig * 0.25, LFDNoise3.kr(modSpeed) * pan, trig * 0.25, LFDNoise1.kr(modSpeed) * pan)});
```

Comment:
`Pan4.ar` has the follwowing arguments: `in, xpos = 0.0, ypos = 0.0, level = 1.0;` I'm not sure what you are trying to do here.

Why not `Pan2.ar( sig, LFDNoise3.kr(0.05) );` ?